app/assets/stylesheets/content/work_package_details/_watchers_tab.sass

```diff
@@ -38,6 +36,9 @@
   input[type='text']
     width: 100%
```

Comment:
Do we still need all those styles @HDinger. Especially the select2/ui-select once I assume to be unnecessary by now.